main.go

```diff
@@ -1,158 +1,197 @@
 package main
 
 import (
-	"bytes"
-	"crypto/tls"
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"net/http"
-	"net/url"
 	"os"
-
-	"github.com/drone/drone-go/drone"
-	"github.com/drone/drone-go/plugin"
-	"github.com/drone/drone-go/template"
+	"log"
+	"github.com/urfave/cli"
 )
 
 const (
 	respFormat      = "Webhook %d\n  URL: %s\n  RESPONSE STATUS: %s\n  RESPONSE BODY: %s\n"
 	debugRespFormat = "Webhook %d\n  URL: %s\n  METHOD: %s\n  HEADERS: %s\n  REQUEST BODY: %s\n  RESPONSE STATUS: %s\n  RESPONSE BODY: %s\n"
 )
-
-var (
-	buildCommit string
-)
+var build string
 
 func main() {
-	fmt.Printf("Drone Webhook Plugin built from %s\n", buildCommit)
-
-	system := drone.System{}
-	repo := drone.Repo{}
-	build := drone.Build{}
-	vargs := Params{}
-
-	plugin.Param("system", &system)
-	plugin.Param("repo", &repo)
-	plugin.Param("build", &build)
-	plugin.Param("vargs", &vargs)
-	plugin.MustParse()
-
-	if vargs.Method == "" {
-		vargs.Method = "POST"
-	}
-
-	if vargs.ContentType == "" {
-		vargs.ContentType = "application/json"
+	app := cli.NewApp()
+	app.Name = "rancher publish"
```

Comment:
Rancher? :)

Reply:
ah, not sure how that got in there. Stupid copy and paste I'm sure. Thanks for catching that